Sources/Subscription/Flows/Models/SubscriptionOptions.swift

```diff
@@ -17,27 +17,41 @@
 //
 
 import Foundation
+import Networking
 
 public struct SubscriptionOptions: Encodable, Equatable {
+    struct Feature: Encodable, Equatable {
+        let name: SubscriptionEntitlement
+    }
+
     let platform: SubscriptionPlatformName
     let options: [SubscriptionOption]
-    let features: [SubscriptionFeature]
+    /// The available features in the subscription based on the country and feature flags. Not based on user entitlements
+    let features: [SubscriptionOptions.Feature]
+
+    public init(platform: SubscriptionPlatformName, options: [SubscriptionOption], availableEntitlements: [SubscriptionEntitlement]) {
```

Comment:
What is the reason for changing the naming here from `features` to `availableEntitlements`. This is confusing and against the naming conventions we agreed upon: 
- entitlements are permissions granted to user's account for accessing features (e.g. account has entitlement  to use VPN)
- features are services bundled together as a subscription plan, set of features defines the complete set of entitlements user is granted when subscribing

https://app.asana.com/0/1208524871249522/1208561749459707/f
https://app.asana.com/0/1208524871249522/1208561749459707/f

Reply:
The reasons are the following: 

The new source of truth for both user entitlements and subscription entitlements is a single function in SubscriptionManager, this function needs to return "something" that explains what features are available and if they are available for the specific user, this something natural name is Feature (entitlement + enabled or not), I tried to invent something different but the result was confusing. 
How we obtain entitlements and features has changed a lot, getting stuck on the previous naming convention didn't make much sense.

The data we get from the backend is identical between user entitlement and subscription entitlement, they are all entitlement so I maintained the same approach, the moment we merge the two concept we get a SubscriptionFeature

The old Feature and Entitlement struct with `var product` was verbose for no reason, the only thingh we care of is the entitlement

The important concept here is that all of these entitlements and where they come from are internal implementation details, the only externally visible object is
```
public struct SubscriptionFeature: Equatable, CustomDebugStringConvertible {
    public var entitlement: SubscriptionEntitlement
    public var enabled: Bool
    ...
}
```